### PR TITLE
feat: move "basePath" /qod/v0 to "url"-property

### LINKED ISSUE #251: Update the "server"-section to align with other CAMARA specifications
**Current description**
```yaml
servers:
  - url: "{apiRoot}/{basePath}"
    variables:
      apiRoot:
        default: http://localhost:9091
        description: API root
      basePath:
        default: qod/v0
        description: Base path for the QoD API
```
**Expected behavior**

```yaml
servers:
  - url: "{apiRoot}/qod/v0"
    variables:
      apiRoot:
        default: http://localhost:9091
        description: API root
```

**Additional context**
Related to https://github.com/camaraproject/NumberVerification/issues/77

### PR DESCRIPTION
**Pull request by maxl2287**
#### What type of PR is this?

Add one of the following kinds:

* correction
* enhancement/feature

#### What this PR does / why we need it:
Remove the basepath and move this to the "url"-property.

#### Which issue(s) this PR fixes:

Fixes #251

### CODE REVIEW

File: code/API_definitions/qod-api.yaml [hunk 1]
```diff
@@ -73,14 +73,11 @@ externalDocs:
 security:
   - oAuth2ClientCredentials: []
 servers:
-  - url: "{apiRoot}/{basePath}"
+  - url: "{apiRoot}/qod/v0"
     variables:
       apiRoot:
         default: http://localhost:9091
         description: API root
```
Comment (hdamker) [inline code/API_definitions/qod-api.yaml:80]: Within last QoD call there was a request to clarify that apiRoot could be also include a path after the host. A proposal for that:

```suggestion
        description: API root, defined by the service provider, e.g. `api.example.com` or `api.example.com/somepath`
```

**Comment by maxl2287** [comment]
@hdamker & @jlurien: 
Added the suggestion of Herbert.
Please review again.

**Comment by jlurien** [comment]
LGTM

**Review by RandyLevensalor — APPROVED** [review]
LGTM

**Review by hdamker — APPROVED** [review]

**Pull request merged**